### DISCUSSION TITLE
BIPES website updates

### DISCUSSION THREAD

**Discussion by rafaelaroca** [BIPES/BIPES#93] [Announcements]
We have made some updates to BIPES website and included a new page (features):

https://bipes.net.br/wp/features/

Hope they are useful :-)

**Comment by SHAO-QUAN-LIN** [comment]
May I ask how should I send hexadecimal command to Arduino over UART transfer in bipes_offline?

**Reply by rafaelaroca** [reply]
Hello. It depends! If Arduino is connected to the default UART (same as used by the USB), you can simply use print with \x and the HEX code you want:

![image](https://user-images.githubusercontent.com/31751413/148694643-175410fd-eacd-4193-9e0f-9c3943bb640b.png)


>>> print('\x0A')

**Reply by SHAO-QUAN-LIN** [reply]
@rafaelaroca I find some problem when I send data to Arduino:
1. No matter what data I send,it always send "05" in prefix and send "0d 0d 0d 04"in suffix.
2. When I send some Hexadecimal，it doesn't convert the collect ascii code like:
C8 = C3 88
DC = C3 99
D0 = C3 90
E4 = C3 A4

For details, see pictures:
![1](https://user-images.githubusercontent.com/97021002/148734474-8638e1ae-37da-4282-be1b-44882e9f8068.png)
(picture1. The code in generator_stubs.js)
![3](https://user-images.githubusercontent.com/97021002/148734608-da3997ea-03bb-4990-994e-e25a19ea8381.png)
(picture2. Result in BIPES interface)
![2](https://user-images.githubusercontent.com/97021002/148734479-59546369-9af0-491e-b835-5b62077525e0.png)
(picture3. The format of the code)

**Comment by jibeer** [comment]
> We have made some updates to BIPES website and included a new page (features):
> 
> https://bipes.net.br/wp/features/
> 
> Hope they are useful :-)

Hello all !
A huge thumb up for the whole team ! What a project ! Everything I was looking is here : multi hardware, blocky based but python as well, online or offline, no messy install but just a simple folder, MQTT and easyMQTT, dashboards, sharings, extensible.....What a beauty ! 
Honestly, all goals already reached for me with BIPES, but can't wait to see where how far you will be able to drive us.
All the best from France :)

**Reply by rafaelaroca** [reply]
Hello @jibeer,

We thank you very much for the feedback. We are very happy that you like BIPES as we do :-) 

We have a BIPES eBook to be released over the next few days, and @JorgeGMarques is working on a new base for BIPES, to make customizations for multi-language, multi-board, multi-blocks easier!

best regards from Brasil! :-)
